extensions/cornerstone/src/init.js

```diff
@@ -90,6 +90,7 @@ export default function init({ servicesManager, configuration }) {
 
   /* Add extension tools configuration here. */
   const internalToolsConfig = {
+    /* TODO ArrowAnnotate input
```

Comment:
👍  Thanks for commenting out so default prompt works

Reply:
:+1: We have an issue for the other prompt somewhere in JIRA.